bin/hyperspy

```diff
@@ -208,4 +214,10 @@ else:
         sys.argv.append('--pylab=inline')
     elif add_pylab is True:
         sys.argv.append('--pylab=%s' % toolkit.replace("qt4", "qt"))
+
+    if ipy_version >= LooseVersion('3.0.0') and args.environment == "notebook":
```

Comment:
This will launch the notebook with everything but matplotlib correctly loaded which may not be what we want given that we seem to agree in that in the notebook all the imports should be explicit. Wouldn't it be better not to copy the `startup` folder if IPython >= 3.0 and to actually delete `default_imports.py` if present (e.g. the user might have upgraded)? Also I would suggest to rename `default_imports.py` to `default_hspy_imports.py`. In this case, I think that the users would also appreciate to get a warning in the notebook explaining that they have to import manually.